cmd/seid/cmd/root.go

```diff
@@ -63,9 +61,6 @@ func (s *rootOptions) apply(options ...Option) { //nolint:unused // I figure thi
 
 // NewRootCmd creates a new root command for a Cosmos SDK application
 func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
-	go func() {
-		http.ListenAndServe(":6060", nil)
-	}()
```

Comment:
Already exposed in sei-cosmos 
https://github.com/sei-protocol/sei-cosmos/blob/main/server/start.go#L131-L139